libraries-bom-protobuf3/pom.xml

```
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0                              http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>libraries-bom-protobuf3</artifactId>
+  <version>0.0.1-SNAPSHOT</version><!-- {x-version-update:libraries-bom-protobuf3:current} -->
+  <packaging>pom</packaging>
+
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>libraries-bom</artifactId>
+    <version>26.49.0-SNAPSHOT</version><!-- {x-version-update:libraries-bom:current} -->
+    <relativePath>../libraries-bom</relativePath>
+  </parent>
+
+  <name>Google Cloud Platform Supported Libraries Protobuf3</name>
+  <description>
+    A compatible set of Google Cloud open source libraries. This BOM is compatible with protobuf-java 3.x versions.
+    Document: https://cloud.google.com/java/docs/bom
+  </description>
+  <url>https://cloud.google.com/java/docs/bom</url>
+  <organization>
+    <name>Google LLC</name>
+    <url>https://cloud.google.com</url>
+  </organization>
+  <inceptionYear>2024</inceptionYear>
+  <developers>
+    <developer>
+      <name>Cloud Java team</name>
+    </developer>
+  </developers>
+
+  <distributionManagement>
+    <snapshotRepository>
+      <id>sonatype-nexus-snapshots</id>
+      <url>https://google.oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <repository>
+      <id>sonatype-nexus-staging</id>
+      <url>https://google.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+  </distributionManagement>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.protobuf</groupId>
+        <artifactId>protobuf-bom</artifactId>
+        <version>3.25.5</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <profiles>
+    <profile>
+      <id>release</id>
+      <activation>
+        <property>
+          <name>performRelease</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-gpg-plugin</artifactId>
+            <version>3.2.5</version>
```

nit.  To align with version in libraries-bom in main branch. https://github.com/googleapis/java-cloud-bom/blob/a98c340a14edf707d6622faea5e01c8ab53f8322/libraries-bom/pom.xml#L121
```suggestion
            <version>3.2.7</version>
```